server/lib/item-transform.js

```diff
@@ -0,0 +1,37 @@
+const get = require('lodash.get');
+
+const PRICE_MATCH = '[PRICE]';
+
+module.exports.itemTransform = item => {
+	item.formatted = {};
+
+	if (item.secondaryPricing && item.secondaryPricing.length > 0) {
+		const { copyPricePath, copyTemplate } = item.secondaryPricing[0];
+		const secondaryPriceValue = get(item, copyPricePath);
+
+		item.formatted.secondaryPricing = copyTemplate.replace(PRICE_MATCH, secondaryPriceValue);
+		delete item.secondaryPricing;
```

Comment:
why does this need deleting?

Reply:
This is modifying a web response, which is then picked up by the `amp` frontend and rendered to the user (the request to the API is made client side) - deleting a non-used property is helpful in a way that it's reducing the bitesize of the response, plus making the frontend behaviour more strict and reduces the need for several `if` statements, that are very much data oriented, where `mustache` via `amp` has quite limited functionality in terms of interpolation

Reply:
is it possible to build an object literal that is returned from this function instead of modifying the input?

Reply:
Done